docs/en/observability/apm/api/index.asciidoc

```diff
@@ -0,0 +1,12 @@
+[[apm-apis]]
+= APM APIs
+
+// TO DO: Make this more helpful?
```

Comment:
The APM Server APIs are mainly intake APIs that accept data from APM agents. There is also the agent config API, but again, this is only used by apm agents. These endpoints are used primarily (perhaps entirely?) by APM agent developers—not by end users. Most users will have no interest in these.

cc @axw in case I'm wrong ⬆️ 

The APM UI APIs, on the other hand, are the APIs that users will want to use to update config, add annotations, upload source maps, etc.